src/atsu/src/com/vhf/VHF.ts

```diff
@@ -0,0 +1,310 @@
+//  Copyright (c) 2021 FlyByWire Simulations
+//  SPDX-License-Identifier: GPL-3.0
+
+import { ATC } from '@flybywiresim/api-client';
+import { FmgcFlightPhase } from '@shared/flightphase';
+import { NXDataStore } from '@shared/persistence';
+import { DatalinkConfiguration, DatalinkProviders, MaxSearchRange, OwnAircraft, VdlMaxDatarate } from './Common';
+
+// worldwide international airports
+// assumptions: international airports provide VHDL communication (i.e. USA)
+// not perfectly realistic, but realistic enough for a frequency occupancy calculation
+const VhfDatalinkAirports: string[] = [
```

Comment:
I'm a bit wary of us manually maintaining this list. I was looking at what the navdata gives us (it would've been nice if there's a flag "international airport"), but other than guessing based on runway lengths or something (which has it's own issues) I can't really think of a better way.

Reply:
Me neither, but we don't have a better alternative atm.